### PR TITLE
Begins work on filters sections. But also more styling and proxy server prep.

### PR DESCRIPTION
**Pull request by jayrbarr**

### CODE REVIEW

File: client/components/App.jsx [hunk 1]
```diff
@@ -13,7 +17,6 @@ const Grid = styled.div`
   padding: 20px;
 `;
 
-
 const App = (props) => {
   const REVIEWS_PER_PAGE = 5;
```
Comment (JasperC1999) [inline client/components/App.jsx:22]: Why do you need this variable if it's always 5?

File: client/components/App.jsx [hunk 2]
```diff
@@ -34,23 +34,39 @@ const App = (props) => {
     return id;
   };
 
+  const getFromApi = (pathString, cb) => {
+    fetch(pathString, {
+      method: 'GET',
+      headers: { 'Content-Type': 'application/json' },
+    })
+      .then((resp) => resp.json())
+      .then((data) => cb(data));
```
Comment (JasperC1999) [inline client/components/App.jsx:44]: why fetch instead of $.ajax?

File: server/server.js [hunk 3]
```diff
@@ -19,4 +19,14 @@ app.get('/:id/:sort/:order/:offset/:numPerPage', ({ params }, res) => {
...
+app.get('/count/:id', ({ params: { id } }, res) => {
+  db.getCount(id, (err, data) => {
+    if (err) {
+      res.sendStatus(500);
+    } else {
+      res.send(data);
+    }
+  });
+});
+
```
Comment (JasperC1999) [inline server/server.js:36]: Why is count it's own endpoint?

File: client/components/App.jsx (hunk 1 above)
Comment (jayrbarr) [reply client/components/App.jsx:22]: I want to add a feature where that changes. How many reviews per page, e.g.

Comment (jayrbarr) [reply client/components/App.jsx:22]: I want to add a feature where that changes. How many reviews per page, e.g.

Comment (jayrbarr) [reply client/components/App.jsx:22]: I want to add a feature where that changes. How many reviews per page, e.g.

File: client/components/App.jsx (hunk 2 above)
Comment (jayrbarr) [reply client/components/App.jsx:44]: Either would work. Fetch meant I didn't have to add jQuery.

File: server/server.js (hunk 3 above)
Comment (jayrbarr) [reply server/server.js:36]: I finally broke down and made a second one. I'll probably refactor it to use the first one.

**Pull request merged**